starter-core/src/test/groovy/io/micronaut/starter/feature/awslambdacustomruntime/AwsLambdaCustomRuntimeSpec.groovy

```diff
@@ -29,6 +30,17 @@ class AwsLambdaCustomRuntimeSpec extends ApplicationContextSpec  implements Comm
         readme.contains("https://micronaut-projects.github.io/micronaut-aws/latest/guide/index.html#lambdaCustomRuntimes")
     }
 
+    @Issue("https://github.com/micronaut-projects/micronaut-starter/issues/723")
+    void 'test readme.md with feature aws-lambda-custom-runtime and graalvm contains extra documentation'() {
+        when:
+        def output = generate(['aws-lambda-custom-runtime', 'graalvm'])
+        def readme = output["README.md"]
+
+        then:
+        readme
+        readme.contains("./gradlew buildNativeLambda -Pmicronaut.runtime=lambda")
```

Comment:
`-Pmicronaut.runtime=lambda` should not be necessary right

Reply:
Same as before